apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -94,6 +97,10 @@ export function InstructorQuestionSettings({
             course: resLocals.course,
             urlPrefix: resLocals.urlPrefix,
           })}
+          <link
+            href="https://cdn.jsdelivr.net/npm/tom-select@2.3.1/dist/css/tom-select.css"
+            rel="stylesheet"
+          />
```

Comment:
We don't ever want to pull from third-party CDNs since those will be blocked in CBTF environments, so this needs to pull from `node_modules`. You can see an example of this here:

https://github.com/PrairieLearn/PrairieLearn/blob/36d38b6ead8d2108061c60d21610858f0adb7ef1/apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts#L84-L87

Note that `tom-select` also includes Bootstrap-specific styles (https://github.com/orchidjs/tom-select/blob/master/src/scss/tom-select.bootstrap4.scss and https://github.com/orchidjs/tom-select/blob/master/src/scss/tom-select.bootstrap5.scss), which we definitely want to use here. I believe this will resolve the issue that you pointed out with the caret being missing. These can be loaded with something like `nodeModulesAssetPath('tom-select/dist/css/tom-select.bootstrap5.css')`. You can check `res.locals.use_bootstrap_4` to know whether to use the Bootstrap 4 or Bootstrap 5 styles.

Reply:
This is great, I didn't realize that the css files were included. Updated to include these as described above in [98f2dd5](https://github.com/PrairieLearn/PrairieLearn/pull/10868/commits/98f2dd558c3f286228530e2a7899b8db990394be).